src/Accounts/Accounts/Properties/Resources.resx

```diff
@@ -622,4 +622,14 @@
   <data name="TypedSubscriptionNotNumber" xml:space="preserve">
     <value>Typed subscription is not a number.</value>
   </data>
+  <data name="AnnouncementsHeader" xml:space="preserve">
+    <value>[Announcements]
```

Comment:
Is the line break after this intentional?

Reply:
nice catch, removed